www/Public/scss/components/_frontNav.scss

```diff
@@ -0,0 +1,37 @@
+
+.navbar {
+  overflow: hidden;
+  background-color: #001f3f; 
+
+ a {
+    float: left;
+    display: block;
+    color: white;
+    text-align: center;
+    padding: 14px 20px;
+    text-decoration: none;
+    &:hover {
+      background-color: #555; 
+      color: white;
+    }
+  }
+}
+
+
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  background-color: #f4f4f4;
+}
+
+.content {
+  padding: 20px;
+}
+
+
+h1 {
+  color: #001f3f; 
```

Comment:
Utilise le module couleur natif à sass comme fait dans le cours par le prof, tu as les variables déjà définies dans le fichier et tu peux en rajouter si tu as besoin

Reply:
C'est noté